### ISSUE TITLE
ansible-vault syntax has changed and is incompatible with ansible-tools

### ISSUE THREAD

**Issue by anttilaiti** [lvillani/ansible-tools#2]
Ansible-vault syntax has changed and is incompatible with ansible-tools. Nowadays it expects the syntax `ansible-vault edit --vault-password-file=/usr/local/bin/ansible-vault-helpers secret.yml`. `vaultify` in ansible-tools tries to use the old format `ansible-vault --vault-password-file=/usr/local/bin/ansible-vault-helper edit secret.yml`, which causes this error:

```
usage: ansible-vault [-h] [--version] [-v] {create,decrypt,edit,view,encrypt,encrypt_string,rekey} ...
ansible-vault: error: unrecognized arguments: --vault-password-file=/usr/local/bin/ansible-vault-helper --vault-password-file=/usr/local/bin/ansible-vault-helper
```

Other commands like `ansible-playbook` still work.

**Comment by anttilaiti** [comment]
I'll create a pull request soon.

**Comment by anttilaiti** [comment]
And here we go: https://github.com/lvillani/ansible-tools/pull/3

**Comment by lvillani** [comment]
The fix for this should be included in 0.1.2, released yesterday. Thanks again for submitting the patch! :+1:
